lib/pkp/js/functions/cern-functions.js

```diff
@@ -144,6 +172,163 @@ function removeOnMobile() {
+		datepicker.datepicker();
+		if(year != "")
+			datepicker.datepicker("setDate", new Date(year, month, day));
+		datepicker.change(function(){
+			afterDatePickerChange(datepicker, selectDay, selectMonth, selectYear)
+		});
+	});
+}
+
+/**
+ * This function change the selects of day, month and year when its datepicker
+ * change to a correct value, and remove them if the datepicker change to a wrong value
+ * @param datepicker
+ * @param selectDay
+ * @param selectMonth
+ * @param selectYear
+ */
+function afterDatePickerChange(datepicker, selectDay, selectMonth, selectYear) {
+	var date = $(datepicker).val(); //dd-mm-yyyy
+	if(date != undefined) {
+		var dateArray = date.split("-");
+		selectDay.selectedIndex = parseInt(dateArray[0]);
+		selectMonth.selectedIndex = parseInt(dateArray[1]);
+		var founded = false
+		var position = 0;
+		$(selectYear.options).each(function() {
+			if(!founded) {
+				if($(this).val() == dateArray[2]) {
+					selectYear.selectedIndex = position;
+				} else {
+					position++;
+				}
+			}
+		});
+		if(!founded) {
+			//If it was none of the years in the select, we select the first option
+			if(position > selectYear.options.length) {
+				selectYear.selectedIndex = 1;
+			} 
+		}
+	} else { //Default, all empty
+		selectDay.selectedIndex = 0;
+		selectMonth.selectedIndex = 0;
+		selectYear.selectedIndex = 0;
+	}
+}
+
+/**
+ * If the var alternativeTitle is instanciated it will overwrite the title of the page
+ */
+function changeTitle() {
+	if(typeof alternativeTitle != "undefined")
+		$("h2").first().html(alternativeTitle);
+}
+
+/**
+ * This function launch an action if the user confirm the message
+ * @param messageToConfirm This is the message that the user needs to confirm
+ * @param actionAfterConfirm This is the action that is going to be performed if confirmed
+ * @param stopPropagationEvent If this parameter is true, we stop the event that caused this function 
+ */
+function cernConfirm(messageToConfirm, actionAfterConfirm, stopPropagationEvent, event) {
+	if(stopPropagationEvent) {
+		if (event.stopPropagation){
+			event.stopPropagation();
+		}
+	    else if(window.event){
+	    	window.event.cancelBubble=true;
+	    }
+	}
+	if(confirm(messageToConfirm)) {
+		eval(actionAfterConfirm);
+	}
+}
+
+
+/**
+ * This function search the elements that has the class error placed by ojs, and add to the
+ * input in the next element the class has-error from bootstrap 
+ */
+function showErrorsOnPage() {
+	$(".error").each(function() {
+			$(this).parent().next().addClass("has-error");
+	});
+}
+
+/**
+ * This function hide all the checkbox and show the checklist-all if this exists
+ */
+function submissionCheckbox() {
+	var checkbox = document.getElementById("checklist-all");
+	if(typeof checkbox != 'undefined') {
+		var parent = $(checkbox).parent();
+		$(parent).show();
+		$(parent).addClass("col-xs-1");
+		$(parent).next().addClass("col-xs-11");
+		if($( document ).width() >= 990) {
```

Comment:
Can you move this line to media query (it will be easier to change it in the future) ?